Modules/Devices/test/Devices.Application.Tests/Tests/Tiers/Commands/DeleteTier/HandlerTests.cs

```diff
@@ -58,6 +59,26 @@ public async Task Cannot_delete_tier_if_CanBeDeleted_returns_an_error()
         A.CallTo(() => _tiersRepository.Remove(tier)).MustNotHaveHappened();
     }
 
+    [Fact]
+    public void Cannot_delete_queued_for_deletion_tier()
```

Comment:
Actually, this test isn't even necessary. It's already covered by the one above. When you were talking about adding a test for it, I thought you wanted to add a test for the Tier.CanBeDeleted method.